admin/data/sql/migrations/20200318133108.do.drop-spcreatechecks.sql

```diff
@@ -0,0 +1 @@
+DROP PROCEDURE IF EXISTS [mtc_admin].[spCreateChecks];
```

Comment:
I'm surprised this is needed - you are using the CREATE OR ALTER statement for the new definition.

Reply:
i would leave it in, as it makes the operation fail safe.  although there is the argument 'surely it should fail if it's not there as there would be a wider issue?'

Reply:
It won't fail if it already exists or if it doesn't exist - it's using CREATE OR ALTER PROCEDURE